manifests/svnrepo.pp

```diff
@@ -66,5 +67,12 @@
     File[$create_path]{
       mode => $mode,
     }
-  } 
+  }
+  # if selinux is enabled and there is a required context apply it
+  if $selinux_context and str2bool($::selinux) {
+    exec {"set_${name}_repo_selinux_context":
+      command     => "semanage fcontext -a -t ${selinux_context} \"${$create_path}(/.*)?\" && restorecon -R -v ${$create_path}",
+      require     => File[$create_path],
```

Comment:
Actually, we could also make it just refreshonly, because we likely need to do that only after we did the checkout.